tests/ToggleGroup.test.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import ToggleButton from '../src/ui/ToggleButton';
+import ToggleGroup from '../src/ui/ToggleGroup';
+
+const renderComponent = () => {
+  return render(
+    <ToggleGroup defaultValue="sedona">
+      <ToggleButton value="sedona">Sedona</ToggleButton>
+      <ToggleButton value="santa_fe">Santa Fe</ToggleButton>
+      <ToggleButton value="las_cruces">Las Cruces</ToggleButton>
+    </ToggleGroup>
+  );
+};
+
+describe('render toggle group component', () => {
+  test('radiogroup should be in the document', () => {
+    const { getByRole } = renderComponent();
+    expect(getByRole('radiogroup')).toBeInTheDocument();
+  });
+
+  test('toggle group should contain all radio buttons', () => {
+    const { queryAllByRole } = renderComponent();
+    expect(queryAllByRole('radio')).toHaveLength(3);
+  });
+
+  test('triggering a click should alter button state', () => {
```

Comment:
The way we will use this component is as a "controlled" component, meaning we will pass it a `value` prop and it should respect that value prop and simply call `onChange` when a click is registered. So this functionality of managing its own state is actually something we don't want. Instead, we should test that:

1. When we pass a `value` to the group, the right button is checked
2. When a button is clicked, `onChange` fires with the right value (you already test this below)